src/main/resources/org/jenkinsci/plugins/credentialsbinding/impl/BindingStep/help.html

```diff
@@ -58,6 +58,29 @@
     anyone permitted to configure a job or define Pipeline steps
     is assumed to be trusted to use any credentials in scope however they like.
 </p>
+<p>
+    Beware that certain tools mangle secrets when displaying them.
+    For example, Bash (as opposed to Ubuntu’s plainer Dash) does so with text containing <code>'</code> in echo mode:
+</p>
+<pre><code>$ export PASS=foo"'"bar
+$ env|fgrep PASS
+PASS=foo'bar
+$ sh -c 'set -x; echo $PASS'
++ echo foo'bar
+foo'bar
+$ bash -c 'set -x; echo $PASS'
++ echo 'foo'\''bar'
+foo'bar
+</code></pre>
+<p>
+    Mangled secrets are not detected by Jenkins and will appear in plain text in the log file, in this case as:
+</p>
+<pre><code>+ echo 'foo'\''bar'
+****</code></pre>
+<p>
+    This particular issue can be worked around by turning off echo with <code>set +x</code>,
```

Comment:
Note that [`sh` sets `-xe`](https://github.com/jenkinsci/durable-task-plugin/blob/c8e1d57986ecc857a0b74f4b21da2b64dc27e3f4/src/main/java/org/jenkinsci/plugins/durabletask/BourneShellScript.java#L118), following the example of [freestyle builds](https://github.com/jenkinsci/jenkins/blob/183ca1f95ebc44318194ff53887b876e47d92d34/core/src/main/java/hudson/tasks/Shell.java#L97), which is undesirable when it comes to secrets.